lang/main.json

```diff
@@ -1,21 +1,36 @@
 {
     "addPeople": {
         "add": "Invite",
+        "addContacts": "Invite your contacts",
+        "copyInvite": "Copy meeting invite",
+        "copyLink": "Copy meeting link",
+        "copyStream": "Copy live streaming link",
         "countryNotSupported": "We do not support this destination yet.",
         "countryReminder": "Calling outside the US? Please make sure you start with the country code!",
+        "defaultEmail": "Your Default Email",
         "disabled": "You can't invite people.",
         "failedToAdd": "Failed to add participants",
         "footerText": "Dialing out is disabled.",
+        "googleEmail": "Google Email",
+        "inviteMoreHeader": "You are the only one in the meeting",
+        "inviteMoreMailSubject": "Join {{appName}} meeting",
+        "inviteMorePrompt": "Invite more people",
+        "linkCopied": "Link copied to clipboard",
         "loading": "Searching for people and phone numbers",
         "loadingNumber": "Validating phone number",
         "loadingPeople": "Searching for people to invite",
         "noResults": "No matching search results",
         "noValidNumbers": "Please enter a phone number",
+        "outlookEmail": "Outlook Email",
         "searchNumbers": "Add phone numbers",
         "searchPeople": "Search for people",
         "searchPeopleAndNumbers": "Search for people or add their phone numbers",
+        "shareInvite": "Share meeting invite",
+        "shareLink": "Share the meeting link to invite others",
```

Comment:
What is this?

Reply:
The text above the `CopyLinkSection` component that prompts users to copy the conference URL